lib/src/common/model/users.dart

```diff
@@ -87,6 +87,10 @@ class User {
   /// Last time this [User] was updated.
   DateTime updatedAt;
 
+  /// The username of the twitter account (without leading @)
+  @JsonKey(name: 'twitter_username')
```

Comment:
Since the User class is annotated with `@JsonSerializable(fieldRename: FieldRename.snake)` the annotation here isn't needed.

```suggestion
```